### PR TITLE
Update macOS image on azure pipeline

### LINKED ISSUE #1964: Update macOS image on Azure pipelines
The macOSX 10.13 VM are being retired.
```
##[error]This job was temporarily blocked because it uses a Microsoft hosted agent (MacOS-10.13) that will be permanently removed on March 23rd, 2020.  See https://aka.ms/blocked-hosted-agent for more information.
,##[error]The remote provider was unable to process the request.
```
See https://devblogs.microsoft.com/devops/removing-older-images-in-azure-pipelines-hosted-pools/

We need to replace the 10.13 image with the 10.14 in our azure pipelines

### PR DESCRIPTION
**Pull request by TG1999**
Signed-off-by: TG1999 <tushar.goel.dav@gmail.com>

Fixes #1964

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

### CODE REVIEW

**Comment by TG1999** [comment]
Hi @pombredanne please can you review it :)

**Comment by TG1999** [comment]
Done @majurg

**Comment by TG1999** [comment]
I will rebase the merge

**Comment by TG1999** [comment]
I checked that I am even with the develop branch, so any thoughts how I can solve this fail.

**Comment by majurg** [comment]
What does the error say? There seems to be a broken link.

**Comment by TG1999** [comment]
Yup @majurg

**Comment by TG1999** [comment]
Okay I got it , I have to fix URLs in documentation I think it has been already done in #1977, so should I repeat the same in my PR ?

**Comment by majurg** [comment]
@TG1999 Ah ok, I have just merged that, so if you rebase off develop and push again everything should be all good.

**Comment by TG1999** [comment]
Done @majurg , rebasing done

**Comment by TG1999** [comment]
Hi @majurg , one more broken URL 
https://www.divio.com/blog/documentation/ -> 
https://docs.divio.com/en/latest/

**Comment by TG1999** [comment]
Hey @majurg, https://travis-ci.org/github/nexB/scancode-toolkit/jobs/664775519?utm_medium=notification&utm_source=github_status this one is failing, can you have a look at it once

File: CHANGELOG.rst [hunk 1]
```diff
@@ -4,7 +4,8 @@ Changelog
 Outputs:
  - The "headers" attribute in JSON outputs now contains a 'duration' field. #1942
 
-
+OS support:
+ - Remove 10.13 macOS image from azure pipeline
```
Comment (majurg) [inline CHANGELOG.rst]: Lets add "Add 10.15 macOS tests to azure-pipelines" here

**Comment by TG1999** [comment]
Done @majurg , thanks for your guidance :)

**Pull request merged**